package.json

```diff
@@ -216,7 +217,8 @@
     "type-detect": "^4.0.8",
     "uuid": "3.3.2",
     "whatwg-fetch": "^3.0.0",
-    "yauzl": "^2.10.0"
+    "yauzl": "^2.10.0",
+    "@aws-sdk/credential-providers": "3.204.0"
```

Comment:
What is the risk of using `aws-sdk@2` with `aws-sdk/credential-providers@3`?

Reply:
will remove `aws-sdk/credential-providers@3`, opensearch client does support using sdk2 to use SigV4